src/components/StudyRoom/StudyRoomChat/Sockets.js

```diff
@@ -1,9 +1,6 @@
 import io from "socket.io-client"
 
 const connectionOptions =  {
```

Comment:
this is out of the scope of the bugfix privacy toggle

Reply:
its a security issue. Was calling undocumented props.